packages/remix-server-runtime/reexport.ts

```diff
@@ -1,4 +1,10 @@
-export type { ErrorResponse } from "@remix-run/router";
+export type {
+  ActionFunction,
+  ActionFunctionArgs,
+  ErrorResponse,
+  LoaderFunction,
+  LoaderFunctionArgs,
+} from "@remix-run/router";
```

Comment:
We should keep exporting these from `./routeModules`

Reply:
yeah I realized I had them wrong locally when I tested on a playground app - pushing that up shortly

Reply:
As long as it's merged _before_ v2 releases as a stable version, I'm fine with that

As removing them would be a breaking change as well